### PR TITLE
e2e: validate histogram bucket

### PR DESCRIPTION
**Pull request by zirain**
e2e: validate histogram bucket

### CODE REVIEW

File: tests/integration/telemetry/common/stats.go [hunk 1]
```diff
@@ -138,6 +144,10 @@ func TestStatsFilter(t *testing.T, feature features.Feature) {
 							return err
 						}
 
+						if err := ValidateBucket(c, prom, "a", expectedBuckets); err != nil {
```
Comment (howardjohn) [inline tests/integration/telemetry/common/stats.go:147]: nit: `"a"` can be clt.Config().Service()? Otherwise its hard to follow what `"a"` means

File: tests/integration/telemetry/envoyfilter/customizemetrics/customize_metrics_test.go [hunk 2]
```diff
@@ -144,6 +145,9 @@ func TestCustomizeMetrics(t *testing.T) {
 			}
 			util.ValidateMetric(t, cluster, promInst, httpDestinationQuery, 1)
 			util.ValidateMetric(t, cluster, promInst, grpcDestinationQuery, 1)
+			if err := common.ValidateBucket(cluster, promInst, "client", 10); err != nil {
```
Comment (howardjohn) [inline tests/integration/telemetry/envoyfilter/customizemetrics/customize_metrics_test.go:148]: Maybe we can comment where `10` came from?

**Review by kyessenov — APPROVED** [review]
P&T approve

File: tests/integration/telemetry/common/stats.go (hunk 1 above)
Comment (zirain) [reply tests/integration/telemetry/common/stats.go]: updated, PTAL.

**Review by howardjohn — APPROVED** [review]
